### DISCUSSION TITLE
S3 server 的文档在哪里？

### DISCUSSION THREAD

**Discussion by bowencool** [AlistGo/alist#6275] [Q&A] [answered]
我配了，然后不生效、不报错、没日志。想排查无从下手：
<img width="1025" alt="image" src="https://github.com/alist-org/alist/assets/20217146/d9ee7920-ed2d-46b6-b506-ea1b049bbcd5">
<img width="377" alt="image" src="https://github.com/alist-org/alist/assets/20217146/282adfe2-8300-489c-9bd1-69ff96c3b82e">
<img width="918" alt="image" src="https://github.com/alist-org/alist/assets/20217146/dec1d4bf-a1ca-424f-924b-f3a054d57520">

**Comment by pongfcnkl** [comment, marked as answer]
https://deploy-preview-335--alist-doc.netlify.app/zh/guide/advanced/s3.html

**Reply by pongfcnkl** [reply]
另外rclone上传好像无效 建议还是用回webdav吧

**Reply by bowencool** [reply]
我大部分场景用的都是WebDAV，但有一个例外就是 duplicacy，它对 WebDAV 的支持仍在 beta 阶段... 

我试了一下 S3 Server：添加储存成功、获取配置成功、备份成功、恢复成功，非常完美！

FYI：https://blog.bowen.cool/zh/posts/how-to-encrypt-backup-your-data-on-your-nas

**Comment by anwen-anyi** [comment]
- https://alist.nn.ci/zh/guide/advanced/s3.html
- 上面那个的临时文档，正式文档中已经合并了
